docs/tutorials/prowler-app-security-hub-integration.md

```diff
@@ -0,0 +1,244 @@
+
+???+ note
+    AWS Security Hub must be enabled in target regions before configuring this integration. Refer to [AWS Security Hub pricing](https://aws.amazon.com/security-hub/pricing/) for cost information.
+
+## Prerequisites
+
+Before configuring the AWS Security Hub Integration in **Prowler App**, ensure the following requirements are met:
+
+### AWS Security Hub Setup
+
+1. **Enable AWS Security Hub** in at least one AWS region for the target AWS account
+2. **Accept the Prowler Integration** in AWS Security Hub to allow findings import
+3. **Configure IAM Permissions** for `securityhub:BatchImportFindings` and `securityhub:GetFindings`
+
+???+ info "AWS Configuration Required"
+    AWS Security Hub must be properly configured before setting up the integration in **Prowler App**. For detailed instructions on enabling Security Hub and accepting the Prowler integration, refer to the [AWS Security Hub Setup Guide](./aws/securityhub.md).
+
+???+ warning
+    If no region has Security Hub enabled, the integration in **Prowler App** will not work. Security Hub is a regional service - enable it in each region where findings should be received.
+
+## Required Permissions
+
+### IAM Policy
+
+The Security Hub integration requires specific permissions to export findings. The IAM role or user credentials must include:
+
+- `securityhub:BatchImportFindings` - To send findings to Security Hub
+- `securityhub:GetFindings` - To retrieve existing findings for updates
+
+???+ tip "Permission Templates Available"
+    For the complete IAM policy JSON, refer to the [prowler-additions-policy.json](https://github.com/prowler-cloud/prowler/blob/master/permissions/prowler-additions-policy.json) on GitHub.
+
+### Infrastructure as Code Templates
+
+**Prowler App** provides IaC templates to automate IAM role creation with Security Hub permissions.
+
+#### CloudFormation
+
+[**Launch CloudFormation Stack →**](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https%3A%2F%2Fprowler-cloud-public.s3.eu-west-1.amazonaws.com%2Fpermissions%2Ftemplates%2Faws%2Fcloudformation%2Fprowler-scan-role.yml&stackName=Prowler&param_EnableSecurityHubIntegration=true)
+
+Key parameters:
+
+- `EnableSecurityHubIntegration`: Set to `true`
+- `ExternalId`: Use the value from **Prowler App**
+- `SecurityHubIntegrationRegions`: Comma-separated list of regions
+
+#### Terraform
+
+[**View Terraform Module →**](https://github.com/prowler-cloud/prowler/tree/master/permissions/templates/terraform)
+
+Configure in `terraform.tfvars`:
+```hcl
+enable_security_hub_integration = true
+security_hub_integration_regions = ["us-east-1", "eu-west-1"]
+external_id = "your-external-id-from-prowler-app"
+```
```

Comment:
Did you create new templates for the AWS Security Hub integration? I remember that yesterday we decided not to. If you created them let's use them because with the current ones there is no params like: `enable_security_hub_integration` or `security_hub_integration_regions`.